cli/DefaultsConfigurator.hpp

```diff
@@ -51,6 +52,21 @@ class DefaultsConfigurator {
   }
 
   /**
+   * @brief Get the number of available numa sockets.
+   *
+   * @return Number of available numa sockets. Always 1 if the system doesn't
+   *         have libnuma.
+   **/
+  static std::size_t GetNumNUMANodes() {
+  #ifdef QUICKSTEP_HAVE_LIBNUMA
```

Comment:
@cramja For preprocessors, there is no indentations.

https://google.github.io/styleguide/cppguide.html#Preprocessor_Directives